app/preprints/-components/submit/metadata/template.hbs

```diff
@@ -9,6 +9,9 @@
         </h3>
         <div local-class='form-container'>
             <div local-class='input-container'>
+                <label>
+                    {{t 'preprints.submit.step-two.contributors-input'}}
+                </label>
                 <Contributors::Widget
```

Comment:
I think you'll want to wrap this in a `{{#let (unique-id ... as |contributorId|}}` here like you do below to link the `<label>` with the input. I think the widget should get the id like this. `<Contributor::Widget id={{contributorId}}`. (As an aside, this one is kind of weird as this `<label>` doesn't directly correlate to an `<input>` type field, but I think it may still warrant the `unique-id` stuff)